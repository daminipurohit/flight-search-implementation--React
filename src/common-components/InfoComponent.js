import React from 'react';
import PropTypes from 'prop-types';

const InfoComponent = ({ children }) => (
    <div className="info-container">
        <div className="info-content">{children}</div>
    </div>
);

InfoComponent.propTypes = {
    children: PropTypes.node.isRequired
};
export default InfoComponent;
