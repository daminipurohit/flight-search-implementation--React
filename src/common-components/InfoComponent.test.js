import React from 'react';
import { shallow } from 'enzyme';
import InfoComponent from './InfoComponent';

describe('InfoComponent', () => {
    it('should render correctly', () => {
        const component = shallow(<InfoComponent><div>I am child</div></InfoComponent>);
        expect(component).toMatchSnapshot();
    });
});
