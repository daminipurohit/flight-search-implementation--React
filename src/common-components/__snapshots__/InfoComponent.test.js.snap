// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoComponent should render correctly 1`] = `
<div
  className="info-container"
>
  <div
    className="info-content"
  >
    <div>
      I am child
    </div>
  </div>
</div>
`;
